lib/decanter/parser/boolean_parser.rb

```diff
@@ -7,7 +7,7 @@ class BooleanParser < ValueParser
       parser do |val, options|
         raise Decanter::ParseError.new 'Expects a single value' if val.is_a? Array
         next if (val.nil? || val === '')
-        [1, '1'].include?(val) || !!/true/i.match(val.to_s)
+        [1, '1'].include?(val) || !!/^true$/i.match(val.to_s)
```

Comment:
While we're here, can we switch this to use the [boolean method](https://ruby-doc.org/core-2.5.1/Regexp.html#method-i-match-3F) form of `match`?